news/643.feature

```diff
@@ -0,0 +1,2 @@
+New resolvers `oc.dict.keys` and `oc.dict.values` allow extracting the lists of keys and values of config nodes
+
```

Comment:
```suggestion
New resolvers `oc.dict.keys` and `oc.dict.values` provides a list view of the keys or values or a DictConfig node.
```

Reply:
Done in 8a65897 (up to minor typo fixes)